CoreScriptsRoot/CoreScripts/BubbleChat.lua

```diff
@@ -48,6 +50,13 @@ local BubbleColor = {	WHITE = "dub",
 --[[ END OF SCRIPT ENUMS ]]
 
 
+if not NON_CORESCRIPT_MODE then
+	while PlayersService.LocalPlayer == nil do PlayersService.ChildAdded:wait() end		--eww
```

Comment:
I think this code will work, but technically LocalPlayer is property, so it isn't necessary for a child to be added for this property to change.